app/channels/room_channel.rb

```diff
@@ -22,43 +22,56 @@ def speak(data)
     if client_action["join"] || client_action["add-ai-player"]
       player = client_action["add-ai-player"] || current_user
       @game.add_player(player)
-      # start_game(@game) if ActionCable.server.connections.map(&:current_user)
-      #   .count == @game.users.count
     elsif client_action["start-game"]
       start_game(@game)
     elsif @game.started
+      @game = Game.find(@game.id)
```

Comment:
Has something changed in the background since the first time we loaded game? Or why do we re find it here? If we need to refind it, you might think about using `@game.reload` to make it more obvious that it's the same object representation